### ISSUE TITLE
There are two releases-created outputs that are inconsistent 

### ISSUE THREAD

**Issue by kurtisvg** [google-github-actions/release-please-action#689]
### TL;DR

The action currently has two outputs that seem to overlap:

- "releases_created" - [is only set when a release has happened](https://github.com/google-github-actions/release-please-action/blob/af1990728f970ad54c88e130c02a1521988127a2/index.js#L224)
- "release_created" - [is only set when release-type is manifest](https://github.com/google-github-actions/release-please-action/blob/af1990728f970ad54c88e130c02a1521988127a2/index.js#L232)

But both of these are referred to in the documentation interchangably:
- [Outputs ](https://github.com/google-github-actions/release-please-action/blob/af1990728f970ad54c88e130c02a1521988127a2/index.js#L232)only described "releases-created" (and says it will be set to false when a release doesn't happen)
- [Manifest release output](https://github.com/google-github-actions/release-please-action/blob/af1990728f970ad54c88e130c02a1521988127a2/index.js#L232) describes "releases-created" (instead of "release-created").
- [Automated publication to npm example uses "release_created"](https://github.com/google-github-actions/release-please-action/blob/af1990728f970ad54c88e130c02a1521988127a2/index.js#L224) - which I think doesn't work because only "releases_created" will be set

### Expected behavior

At this point, I think both "releases_created" and "release_created" should be set properly to true/false regardless of what type of release it is. 

### Action YAML

```yaml
N/A
```

**Comment by eokten-homex** [comment]
This is definitely creates a little bit confusion. Either the output needs to be set everytime or the documentation should be changed to reflect that.
